### PR TITLE
Started fixing settings page

### PR DESCRIPTION
**Pull request by disjfa**
So i made a simple update, just to save my profile for a start.

Note, also made a change to the webpack config in config/index to build the site to /realworld-vue/ so i can just deploy it to a gh-pages page (https://disjfa.github.io/realworld-vue/) just to deploy. We will need to change/setup the base. This is just a reminder.

Added route after saving
Added put to apiclient
Setup babelrc for building
Added gh-pages for deployment to gh-pages

### CODE REVIEW

**Comment by disjfa** [comment]
Ok, so i also added a basic profile page. I was just typing, so why not. Did set up some routes for easy managing my articles and faforited articles as an example.

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (vilsbole) [inline .babelrc]: Separate commit please

Comment (disjfa) [reply .babelrc]: Sure, don't know why tho. But sure!

**Comment by vilsbole** [comment]
Please excuse me for the incomplete review, my coworking closed before I could complete the review.

My primary concern is that this PR mingles concerns (gh-pages deploy, stylechanges, feature implementation). The second is that the feature is incomplete (which is fine considering that it's an isolated WIP). It reminds me that I need to specify contributing guidelines (what do you think of #24 ?), and that the linter doesn't check templates.

If you could just make a few changes then we can move forward:
- [ ] respect [single concern](http://blog.ploeh.dk/2015/01/15/10-tips-for-better-pull-requests/) pull request
- [ ] respect standard 2 space indentation

NB. I love that you use child views! It was one of the techniques that was missing in the project.

**Comment by disjfa** [comment]
Thats ok, and i agree with you. I wanted to ask the same question about how to pull request.

Can i ask the hard question and ask if i can just merge this one (or when i finish the profile page), i can nitpick and cherry pick the code and make my day worse by the code. Or i can finish the profile page and my next pull request will use the guidlines.

**Comment by vilsbole** [comment]
By experience I've learned that that code quality increases when we maintain the standard, so I believe that it's in the projects interest not to comprise.

Instead of cherry-picking you could maybe [revert](https://stackoverflow.com/questions/692246/undo-working-copy-modifications-of-one-file-in-git) the `package.json` file ?

**Comment by disjfa** [comment]
Harsh! Will do.

**Comment by disjfa** [comment]
I made a second pull request for this using contributing guidelines.

**Pull request closed without merging**